sortinghat/core/api.py

```diff
@@ -644,6 +714,56 @@ def delete_domain(ctx, domain_name):
     return domain
 
 
+@django.db.transaction.atomic
+def delete_group(ctx, organization, group_name):
+    """Remove a group from the registry.
+
+    This function removes the given group of an organization from the registry.
+    Deleting a group also deletes all its subgroups in the organization.
+    Both Organization and Group must exist in the registry. Otherwise, function
+    will raise a `NotFoundError` exception.
```

Comment:
You should reword this because the group doesn't need to belong to an organization.

Reply:
Done 👍